.gitignore

```diff
@@ -42,3 +42,4 @@ nvfuser/cmake
 
 # debugging temporaries
 *.log
+foo.bin
```

Comment:
Was this just added accidentally?

Reply:
This is not. Some test generate this, I am not sure, but probably related to serialization or kernel db.